app/src/main/java/com/keylesspalace/tusky/EditProfileActivity.kt

```diff
@@ -94,93 +88,133 @@ class EditProfileActivity : BaseActivity(), Injectable {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_edit_profile)
 
+        viewModel = ViewModelProviders.of(this, viewModelFactory)[EditProfileViewModel::class.java]
+
         setSupportActionBar(toolbar)
         supportActionBar?.run {
```

Comment:
This is not your change now, but inside `run` we don't need variable name

Reply:
yeah but it still is my code :D